### PR TITLE
Add model AC2939

### LINKED ISSUE #38: New purifier model: AC2939
Hello,

Here's the raw data for my purifier model AC2939/10 (powered off):

`{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "0", "pwr": "0", "cl": false, "aqil": 100, "uil": "1", "uaset": "P", "mode": "AG", "pm25": 4, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 1862095, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}`
(DeviceId removed for privacy/security reasons)

The status above shows the "clean filter" at 153 hours, and "replace filter" at 4752 hours.

4 modes are supported: Auto [AG], Turbo [T], Gentle [GT], and Night Mode [S]:

```
{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "0", "pwr": "1", "cl": false, "aqil": 100, "uil": "1", "uaset": "P", "mode": "AG", "pm25": 1, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 2210874, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}

{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "t", "pwr": "1", "cl": false, "aqil": 100, "uil": "1", "uaset": "P", "mode": "T", "pm25": 1, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 2245154, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}

{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "1", "pwr": "1", "cl": false, "aqil": 100, "uil": "1", "uaset": "P", "mode": "GT", "pm25": 1, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 2256826, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}

{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "s", "pwr": "1", "cl": false, "aqil": 100, "uil": "1", "uaset": "P", "mode": "S", "pm25": 1, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 2263005, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}
```

and you can turn the light (uil) off:

```{"name": "AC2939", "type": "AC2939", "modelid": "AC2939/10", "swversion": "Ms2102", "language": "EN", "DeviceVersion": "1.0.2", "om": "s", "pwr": "1", "cl": false, "aqil": 0, "uil": "0", "uaset": "P", "mode": "S", "pm25": 1, "iaql": 1, "aqit": 4, "tvoc": 1, "ddp": "1", "rddp": "1", "err": 0, "fltt1": "A3", "fltt2": "none", "fltsts0": 153, "fltsts1": 4752, "fltsts2": 0, "filna": "0", "filid": "0", "range": "MarsLE", "Runtime": 2266465, "WifiVersion": "AWS_Philips_AIR@62.1", "ProductId": "b97ea7a8170b11eaa1e3061302926720", "DeviceId": "[removed]", "StatusType": "localcontrol", "ConnectType": "Localcontrol"}```

### PR DESCRIPTION
**Pull request by Steltek**
Should fix #38 if I didn't mess it up.

### CODE REVIEW

File: custom_components/philips_airpurifier/const.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 MODEL_AC1214 = "ac1214"
 MODEL_AC2729 = "ac2729"
 MODEL_AC2889 = "ac2889"
+MODEL_AC2889 = "ac2889"
```
Comment (betaboon) [inline custom_components/philips_airpurifier/const.py:12]: one too many :D

**Review by betaboon — COMMENTED** [review]
could you please rebase against master-branch :)

**Comment by Steltek** [comment]
Thanks for catching the extra const. Hope I did the rebasing thing right. (Not too familiar with Git/Github.)

**Comment by betaboon** [comment]
thank you for your contribution

**Pull request merged**